ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -337,29 +341,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-minimal-permission
-- as: aws-ipi-ovn-fips-p2-f28
+- as: aws-ipi-ovn-fips-efs-p2-f28
```

Comment:
Since 4.10, all the ci coverage should be covered in both amd and arm unless the feature is not supported in arm, shall we cover in arm too?

Reply:
Thanks for reminding! We plan to cover the `arm64` with a different config,I'll file 1 separate PR migrate`13_aarch64_IPI on AWS & OVN & STS & EFS`.  Is it resonable for current CI coverage?

Reply:
During the migration, we are not force a 100% equivalent with the ci profile, if EFS does not care about the other configurations, it is okay to integrate EFS on the same configuration as amd. Sometimes that can help us easily compare test results, also easily maintain ci coverage on both arches with copy and paste.

Reply:
Make sense, added arm64 jobs. Could you please help take a look again when you get a chance? Thanks for the clarifying!